src/test/java/com/epam/reportportal/ui/AdditionalTest.java

```diff
@@ -0,0 +1,111 @@
+package com.epam.reportportal.ui;
+
+import com.epam.reportportal.utils.DriverManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class AdditionalTest {
+
+    private final WebDriver driver = DriverManager.getDriver();
+
+    @Test
+    public void resizeElement() throws InterruptedException {
+        driver.get("https://jqueryui.com/resizable/");
+        driver.switchTo().frame(0);
+
+        WebElement resizeableElement = driver.findElement(By.xpath("//div[contains(@class,'ui-resizable-se')]"));
+        resize(resizeableElement, 200, 200);
+        Thread.sleep(1000);
+    }
+
+    public void resize(WebElement elementToResize, int xOffset, int yOffset) {
```

Comment:
the idea was to create a utility with abstraction that can be used anywhere in the code.
(For actions, js scripts)

Reply:
ActionUtils class has been created